docs/duration.md

```diff
@@ -54,7 +54,7 @@ For more detailed information, see the ISO 8601 standard or the [Wikipedia page]
 
 All of the arguments are optional.
 Any missing or `undefined` numerical arguments are taken to be zero, and all non-integer numerical arguments are rounded down to the nearest integer.
```

Comment:
re: "are rounded down to the nearest integer", what does this mean for negative values?  Is it rounded towards zero or towards negative infinity?  Should clarify because "down" is ambiguous now.